src/Entity/IdeasWorkshop/Theme.php

```diff
@@ -62,4 +77,14 @@ public function setEnabled(bool $enabled): void
     {
         $this->enabled = $enabled;
     }
+
+    public function getImagePath(): string
+    {
+        return sprintf('images/themes/%s', $this->getImageName());
```

Comment:
Add `ideas_workshop` to the images path: `images/ideas_workshop/themes/`.

Reply:
done